src/core/http/api/BaseApi.ts

```diff
@@ -43,7 +43,10 @@ export class BaseApi {
             method: method,
             withCredentials: true
         }
-        if (data) config.data = data
+        if (data) {
+            if (method == HTTPMethod.GET) config.params = data
+            else config.data = data
```

Comment:
```suggestion
            if (method == HTTPMethod.GET) {
              config.params = data
            } else {
              config.data = data
            }
```
А вообще стоит учесть что не у GET может быть и тело и параметры